src/Domain/Creators/AlignedLattice.cpp

```diff
@@ -20,36 +21,58 @@ AlignedLattice<VolumeDim, TargetFrame>::AlignedLattice(
     const typename BlockBounds::type block_bounds,
     const typename IsPeriodicIn::type is_periodic_in,
     const typename InitialRefinement::type initial_refinement_levels,
-    const typename InitialGridPoints::type
-        initial_number_of_grid_points) noexcept
+    const typename InitialGridPoints::type initial_number_of_grid_points,
+    const typename BlocksToExclude::type blocks_to_exclude) noexcept
```

Comment:
make non-const

Reply:
This is so that `std::move` actually does something right? Should the other parameters be made non const as well? This can be made an issue, but I'll fix the line I added.